.docker/qgis3-build-deps.dockerfile

```diff
@@ -144,6 +144,10 @@ RUN locale-gen
 
 RUN echo "alias python=python3" >> ~/.bash_aliases
 
+# OTB: download and install otb packages for QGIS tests
+RUN curl -k https://orfeo-toolbox.org/qgis/OTB-Linux64.run -o /tmp/OTB-Linux64.run && sh /tmp/OTB-Linux64.run --target /opt/otb
```

Comment:
@3nids @elpaso is this acceptable?

Reply:
I don't know if this is still an issue. OTB must be installed and if you are worried about time to download, I think it will be faster than apt-get (which is broken on ubuntu last time I checked). A bonus point is it helps otb team to check rc packages against qgis master to make sure descriptor file are okay before qgis and otb releases.

Reply:
I may miss your point, but I'd be against installing from anything but official packages.

Reply:
Ping @3nids  @elpaso -- is this an acceptable inclusion in the docker images?

Reply:
@pcav , no problem with that. These are also official packages from orfeo-toolbox.org. You can find about it here: https://www.orfeo-toolbox.org/download/.

Reply:
The docker images are mostly used for testing.
If the decision is that the OTB provider belongs into core it should be tested and will need to have OTB binaries around for that. And I would prefer to ship these as part of the docker image than having a (hopefully not too often but still sometimes) unstable download in travis.

Reply:
@m-kuhn Can you tell me what you mean by ship otb binaries as part of docker image?. I thought that qgis3-build-deps.dockerfile was reused for development and testing. So it can work with travis and other cases.

Reply:
There are [docker images](https://hub.docker.com/r/qgis/qgis3-build-deps/tags) based on qgis3-build-deps.dockerfile available with the build dependencies for qgis. During travis test runs, containers based on these images are used to compile and test qgis.

Reply:
Is current qgis3-build-deps.dockerfile pushed to latest tag during release or manually.? OTB install can copied from travis by adding `/opt/otb` like the .ccache maybe? I saw that qgis.dockerfile is pushed when running travis-ci with `DOCKER_BUILD_QGIS_IMAGE` https://github.com/qgis/QGIS/blob/master/.ci/travis/linux/script.sh#L21

PS: I am not much familiar with docker.  I appreciate if you can direct me which docker files to update or if you can copy otb install line to correct file.

Thanks in advance

Reply:
It's pushed automatically as soon as it has been merged to a realease or master branch.
To me, the way it's done in here looks good.

Reply:
Thanks for checking this change. Glad that things looks good for docker images for qgis.